askama_shared/src/generator.rs

```diff
@@ -807,24 +807,25 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
         self.handle_ws(ws);
         self.write_buf_writable(buf)?;
         buf.write("let ");
+        self.visit_target(buf, var);
+        buf.writeln(";")
+    }
+
+    fn write_let_needs_let(&self, var: &'a Target<'_>) -> bool {
```

Comment:
This name is not so clear to me. Maybe revisit it or add some documentation about what cases this is intended to handle? In particular, this is quite dissimilar from the other methods that have a name starting with `write_`, so that should be stripped off.

Reply:
Yeah … I need to clean that up. Will do.